5.auth-service/SQL/sql.sql

```diff
@@ -0,0 +1,16 @@
+
+
+CREATE TABLE accounts
```

Comment:
please create one SQL file for all.
under the main folder.
thanks.

Reply:
done